### ISSUE TITLE
Indentation control error

### ISSUE THREAD

**Issue by osimicek** [palantir/tslint#16]
Indentation control in tslint disallows me to write a pretty function calls.

``` javascript
registerController('baseCtrl',
                    BaseCtrl,
                    [
                        "$scope",
                        "$compile",
                        "$http",
                        "$routeParams"
                    ]);
/**
* For lines 4-7 it raises error:
* "Unexpected tab width, expected 4, got 24"
*/
```

OR

``` javascript
show(addr, function(data) {
                alert(data);
            }
    );
/**
* For line 2 it raises error:
* "Unexpected tab width, expected 4, got 16"
*/
```

OR

``` javascript
show(addr,
    (data) =>{
        alert(data);
    },
    "hello",
    "how",
    "are",
    "you"
    );

/**
* For line 3 it raises error:
* "Unexpected tab width, expected 4, got 8"
*/
```
